packages/core/permissions/src/__tests__/permissions.engine.test.ts

```diff
@@ -1,8 +1,14 @@
-'use strict';
-
-const _ = require('lodash');
-const { subject } = require('@casl/ability');
-const permissions = require('..');
+import _ from 'lodash';
```

Comment:
"This module is declared with 'export =', and can only be used with a default import when using the 'esModuleInterop' flag"